Corona-Warn-App/src/main/java/de/rki/coronawarnapp/notification/TestResultAvailableNotificationService.kt

```diff
@@ -1,25 +1,26 @@
 package de.rki.coronawarnapp.notification
 
 import android.content.Context
+import androidx.annotation.IdRes
 import androidx.navigation.NavDeepLinkBuilder
 import de.rki.coronawarnapp.R
 import de.rki.coronawarnapp.coronatest.server.CoronaTestResult
 import de.rki.coronawarnapp.main.CWASettings
 import de.rki.coronawarnapp.ui.main.MainActivity
 import de.rki.coronawarnapp.util.device.ForegroundState
-import de.rki.coronawarnapp.util.di.AppContext
 import de.rki.coronawarnapp.util.notifications.setContentTextExpandable
 import kotlinx.coroutines.flow.first
 import timber.log.Timber
-import javax.inject.Inject
 import javax.inject.Provider
 
-class TestResultAvailableNotificationService @Inject constructor(
-    @AppContext private val context: Context,
+open class TestResultAvailableNotificationService(
+    private val context: Context,
```

Comment:
@AppContext  ?

Reply:
this class won't be injected, only subclasses of it, so no injection annotations needed, i guess?